typedjsonrpc/server.py

```diff
@@ -73,6 +77,9 @@ def _dispatch_request(self, request):
         endpoint, _ = adapter.match()
         if endpoint == self._endpoint:
             return self._dispatch_jsonrpc_request(request)
+        elif endpoint == ROOT_ENDPOINT_NAME:
+            return Response(request.method.__repr__() + " is alive!",
```

Comment:
`repr(request.method)` does the same thing and is the suggested way of doing it.